### PR TITLE
Using service account personal access token for audit logs push

### PR DESCRIPTION
**Pull request by misterzero**

### CODE REVIEW

File: logs/audit-index.csv [hunk 1]
```diff
@@ -13,3 +13,7 @@ ce75011f34cd32746660387a510a8090c3c3a60a, 2022-03-02T21:15:43+00:00
 f7824a0db868c6fb35dfa1406619bf340e25c144, 2022-03-09T23:04:47+00:00
 4e17ef73a670f373b56917ce506ebedc7fa0cb29, 2022-03-09T23:42:57+00:00
 dc886a9ad1beed821bf1305bd364b3865744ab9d,2022-03-11T12:17:06+00:00
+a5437cec13d0cb9ceeba6e02c250a069194f8b75,2022-03-12T04:56:37+00:00
```
Comment (christianpaquin) [inline logs/audit-index.csv:16]: Do you want to add a space between ID and timestamp, as was added in the yaml file.

File: logs/audit-index.csv [hunk 2]
```diff
@@ -13,3 +13,7 @@ ce75011f34cd32746660387a510a8090c3c3a60a, 2022-03-02T21:15:43+00:00
...
+60c28177fd3756049114a0248e28284abf61bdd7,2022-03-13T04:58:28+00:00
+74aecd1383422bb7314efbb5c65bcb81d604444c,2022-03-14T05:12:52+00:00
+77a5958659bd4ca17fce5d44b8c77eb9778f9ff2,2022-03-15T04:56:07+00:00
```
Comment (christianpaquin) [inline logs/audit-index.csv:19]: I see 4 commit events, but only one set in this PR? Will the other be available in the repo?

Comment (misterzero) [reply logs/audit-index.csv:19]: I believe all these commits are included in this PR, can you see them above @christianpaquin ?

Comment (christianpaquin) [reply logs/audit-index.csv]: Ah yeah, I see!

**Pull request merged**